ourjapanlife/findadoc-server
# 52 seed data

Part of #52 

I'm writing the logic around loading some seed data into the database. 

## Regarding the CSV files
I'm sure there will be some comments and thoughts about the contents seed data I've added here. I'd ask that we move those comments to the Google Doc where I am aggregating this information: https://docs.google.com/spreadsheets/d/19hrPp5YvHlEEelpnjMDL6Oh5UwHnST3SoKIudX6zk-M/edit#gid=1175476498

I've also asked in the MVP slack channel if Russ and Aya can add their input regarding medical data. As for the spoken languages, I've reused some information we gathered for speakher.jp

## Regarding the logic changes

- had to add `ts-node` to get the seed function to build

## how to test

1. Get your DB working by running the migration:
```
yarn prisma migrate dev
```

This should automatically run the seed file

OR if you previously ran the migrations:
```
yarn prisma db seed
```

2. View the data by running the prisma explorer: `yarn prisma studio` or if you prefer low level: `psql findadoc`

(If you ever want to[ reset your dev environment for testing](https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-reset) use `yarn prisma migrate reset` 💣 

a. In the prisma explorer you can navigate to http://localhost:5555/ then click on `SpokenLanguages`, which should open up a table of the data that you can interact with (disregard English in there twice, I must have run the seeding on a dirty db)

<img width="923" alt="Prisma_Studio" src="https://user-images.githubusercontent.com/4602369/209818657-a5016ad7-f436-498c-8627-4a96b88638d5.png">
